docs/topics/api/blocklist.rst

```diff
@@ -26,7 +26,6 @@ This endpoint returns an add-on Block from the blocklist, specified by guid or i
     :>json string modified: The date the block was last updated.
     :>json object|null addon_name: The add-on name, if we have details of an add-on matching that guid (See :ref:`translated fields <api-overview-translations>`).
     :>json string guid: The guid of the add-on being blocked.
-    :>json string min_version: The minimum version of the add-on that will be blocked.  "0" is the lowest version, meaning all versions up to max_version will be blocked.  ("0" - "*" would be all versions).
-    :>json string max_version: The maximum version of the add-on that will be blocked.  "*" is the highest version, meaning all versions from min_version will be blocked.  ("0" - "*" would be all versions).
+    :>json string versions[]: The versions of this add-on that are blocked.
```

Comment:
yeah, I just edited the docs - the fields are still present in v5 until I (or someone else) updates frontend and removes them from the API in a follow-up.  Once there are blocks with gaps in the ranges of versions the fields are inaccurate though so I removed them.  I can leave them in the docs for now if you prefer?

Reply:
I had not read the code yet so I was a bit surprised. I guess maybe leave them while it's there, with a warning that we have to migrate? We'll probably do it sooner than later but it might be better to be consistent